### DISCUSSION TITLE
直接使用NTK支持到200K吗？是否在200k的上下文上做过训练？有没有超长文本的评测报告

### DISCUSSION THREAD

**Discussion by lvjianxin** [InternLM/InternLM#679] [Q&A]
### 描述问题

“[QA] LMDeploy 实现了 dynamic ntk，支持长文本外推。使用如下代码，可以把 InternLM2 的文本外推到 200K：”。针对这句话的疑问，请解答下

**Comment by ZwwWayne** [comment]
对，直接使用 dynamic ntk 可以外推到 200K，没有在 200k 语料上训练过，后续的 tech report 会介绍细节，目前可以先参考 https://zhuanlan.zhihu.com/p/678871288

**Reply by datalee** [reply]
考虑过适配YaRN等方法吗？
NTK-RoPE 的效果不见得多好（相比 YaRN、ReRoPE 等）
